### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by KUBAHO3**
In this project:

- Added a desktop version of my portfolio
- added menu detail
- connected menu with the sections
- fixed linter errors
- updated readme file

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -288,3 +310,196 @@ form textarea {
...
+  a {
+    text-decoration-line: none;
+    color: #344563;
+  }
+
+  .container {
+    padding-right: 10%;
+    padding-left: 10%;
+  }
+
```
Comment (zieeco) [inline style.css:329]: - Hey, let's correctly align the form button to the center to match the design as much as we can. see the images below:

*Figma design*
Screenshot from 2023-02-22 16-59-06

*Your design*
Screenshot from 2023-02-22 16-58-46

File: style.css [hunk 2]
```diff
@@ -4,20 +4,37 @@
   font-family: "Poppins", sans-serif;
 }
 
+body {
+  margin: 0 !important;
+}
+
```
Comment (zieeco) [inline style.css]: - Hey, according to the microverse [HTML & CSS requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), you should not overuse the `important` keyword, while you've used it once, please, consider you shouldn't use it at all except you're using a CSS framework. please, kindly try and fix this by not using the keyword in your styles. let's improve on this please :pray: 

- Here, you can add this reset to fix the issue below:

```
*,
*::before,
*::after {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}
```

if you've explicitly used the box-sizing, you can remove it.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @KUBAHO3 ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (KUBAHO3) [reply style.css:329]: Thank you! I haven't noticed that!

Comment (zieeco) [reply style.css:329]: @KUBAHO3, It happens, we forget at times!

File: style.css (hunk 2 above)
Comment (KUBAHO3) [reply style.css]: Solved all required changes! Thanks for review

**Review by osamaashraf6 — APPROVED** [review]
Hi @KUBAHO3 👋 , 
# STATUS: APPROVED! 🥇🟢🟢💯 


Your project is complete!  There is nothing else to say other than... it's time to merge it : ship it: 
Congratulations! 🎉 

## Highlights💯🎈🪄✔️



- [x] Descriptive READme file in a professional way.✔️
- [x] No linter errors. ✔️
- [x] Good usage in correcting GitHub flow.✔️
- [x] Nice design.✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**